### ISSUE TITLE
Progress indicator has DAP violation

### ISSUE THREAD

**Issue by prince737** [carbon-design-system/carbon#5353]
Progress indicator has the following DAP violation: 
`Each form control must have associated label.`

## Environment
Carbon-components: 10.9.3
carbon-components-react: 7.9.3
DAP: 2020 Jan ruleset

## Sandbox
https://codesandbox.io/s/cocky-franklin-86crq

## Screenshot
<img width="1398" alt="Screenshot 2020-02-14 at 12 30 02 PM" src="https://user-images.githubusercontent.com/31027320/74508727-bfe40f00-4f25-11ea-822c-a36e6e993cf4.png">

**Comment by tw15egan** [comment]
Seems like this has been fixed via https://github.com/carbon-design-system/carbon/pull/5125 and will be included in the next release (`10.10.0`), as I am not seeing this DAP violation when running the component locally. 

I am seeing it at `react.carbondesignsystem.com` which is running `10.9.3`. I'm going to close this since it is resolved pending release, but if you still see this after upgrading to `10.10.0` we can reopen and investigate
